docs/releases/2.2.txt

```diff
@@ -333,6 +336,11 @@ backends.
   ``DatabaseIntrospection.get_constraints()`` returns must be normalized by
   ``identifier_converter()``.
 
+* ``DatabaseFeatures.supports_stddev`` has been removed. It was originally
```

Comment:
Maybe `has been removed` -> `is removed` :thinking:

Reply:
It seems this removal would only affect third party database backends that override it -- and if there are such backends, then it would seem to be an argument for keeping the feature, otherwise the backend must use some other hack to skip the test.

Reply:
I checked and none of the official [3rd-party-database-backends](https://docs.djangoproject.com/en/dev/ref/databases/#using-a-3rd-party-database-backend) uses this feature.